pkg/controller/cassandracluster/cassandra_status.go

```diff
@@ -182,6 +187,20 @@ func UpdateStatusIfUpdateResources(cc *api.CassandraCluster, dcRackName string,
 }
 */
 
+//UpdateStatusIfStatefulSetStatusHasChanged updates CassandraCluster Action Status if any pod is not Ready
+func UpdateStatusIfStatefulSetStatusHasChanged(cc *api.CassandraCluster, dcRackName string, storedStatefulSet *appsv1.StatefulSet, status *api.CassandraClusterStatus) bool {
```

Comment:
This one looks dangerous to me. It can't happen during the creation of the cluster, whenever there is an operation at the stfs level etc..
@allamand don't hesitate to weigh in.

Reply:
yes, I didn't take into account the creation of the cluster, 
I will close it.